client/webserver/webserver.go

```diff
@@ -625,6 +627,34 @@ func New(cfg *Config) (*WebServer, error) {
 	return s, nil
 }
 
+// fetchLatestVersion is a helper function to retrieve the latest version of the app
+// from github.
+func (w *WebServer) fetchLatestVersion(ctx context.Context) {
+	ticker := time.NewTicker(1 * time.Minute)
```

Comment:
This is checking way too often. Can it check once when starting up and then once a day? You said yourself we don't update that much.

Reply:
This ticker is for when we did not get a response maybe due to network abnormalities the first time, we will try every 1min until we get our first ok response then the next run is in the next 24hrs

Reply:
Oh ok, It looked like it was checking regardless, will look again.

Reply:
There’s a return statement in the block that hits the GitHub API.